### DISCUSSION TITLE
Edit saved session title?

### DISCUSSION THREAD

**Discussion by Jonas-devel** [sienori/Tab-Session-Manager#868] [Q&A] [answered]
Is there any way to later change the title (name) of a saved session after you've already saved it under a different name? I just installed TSM a little while ago and can't figure out how to do that. Thanks.

**Comment by sienori** [comment, marked as answer]
Click the session name to rename it.
![image](https://user-images.githubusercontent.com/29514424/136994135-62927fa0-a9e9-4dc9-ae8b-f87f7a89e9c2.png)

**Reply by iamhammer** [reply]
Feature request: Add "Rename Session" to the session context menu.
